tests/plugins/premisrw/test_premis.py

```diff
@@ -471,3 +471,77 @@ def test_object_characteristics_extension(self):
             premis_element.xpath(xpath_lookup, namespaces=premis_element.nsmap)[0]
             == embedded_xml
         )
+
+    def test_premis_rights(self):
+        """Test PREMISRights."""
+        PREMIS_RIGHTS_STATEMENT_SAMPLE_DATA = (
```

Comment:
There's no reason this can't be here in the function but I find it is so long it makes it harder to read. I would move it to the class or module level (per PEP8, constants are "usually defined on a  module level"). Again a style choice though.

Reply:
Moved to `test.constants`, which may be worse but now it's consistent!